### ISSUE TITLE
random(b) can return b

### ISSUE THREAD

**Issue by lgto4** [processing-js/processing-js#256]
Related to issue #244, `random(b)` can return `b` when the interval [0 .. b) is sufficiently small.  Here is the most extreme example where the answer is wrong 50% of the time: 
```
var b = 5e-324;
var r = random(b);
println(r + ((r === b) ? " fail!" : " success."));

**Comment by CapacitorSet** [comment]
It seems that this has also been fixed in #269, pinging @Pomax to close this.

By the way, you were right, it appears that indeed commit messages in pull requests do not automatically close issues (just reference them). It only works in one's own repositories.

**Comment by Pomax** [comment]
closing as fixed in 1.6.3
